workers/miflora.py

```diff
@@ -20,8 +18,47 @@ def _setup(self):
       _LOGGER.debug("Adding %s device '%s' (%s)", repr(self), name, mac)
       self.devices[name] = MiFloraPoller(mac, BluepyBackend)
 
+  def config(self):
+    ret = []
+    for name, poller in self.devices.items():
+      ret += self.config_device(name)
+    return ret
+
+  def config_device(self, name):
+    ret = []
+    device={"identifiers": self.format_topic(name, separator="_"),
```

Comment:
Maybe this should have device mac / bt address ?

Reply:
you are definitly right 😁 the mac is the better options here

Reply:
a changed the id generation, add the mac address and updated the example in the pr 😉